packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -370,13 +412,20 @@ export class AnchoredRegion extends FASTElement {
     private pendingReset: boolean = false;
     private currentDirection: Direction = Direction.ltr;
 
+    // defines how big a difference in pixels there must be between states to
```

Comment:
Would exposing this as a public property be useful to app authors?

Reply:
perhaps, but my thinking is that I'd like to delay adding specific api's to control this until we see a need